features/attached_enable.feature

```diff
@@ -119,9 +119,32 @@ Feature: Enable command behaviour when attached to an UA subscription
             See: sudo ua status
             """
 
+    @wip
     @series.focal
-    Scenario Outline: Attached enable of non-container services in a focal lxd container
-        Given a `focal` lxd container with ubuntu-advantage-tools installed
+    @uses.config.machine_type.lxd.vm
+    Scenario: Attached enable of vm-based services in a focal lxd vm
+        Given a `focal` machine with ubuntu-advantage-tools installed
+        When I attach `contract_token` with sudo
+        And I run `ua enable <service> <flag>` as non-root
```

Comment:
I think it is missing the examples table here. When running the tests, I get the following error:

```
Creating behave-test-vm-focal-1593107907758756
Starting behave-test-vm-focal-1593107907758756
Retrying on unexpected cloud-init status stderr:  Error: Failed to connect to lxd-agent
Retrying on unexpected cloud-init status stderr:  Error: Instance is not running
Retrying on unexpected cloud-init status stderr:  Error: Failed to connect to lxd-agent
    Given a `focal` machine with ubuntu-advantage-tools installed ... passed in 146.671s
    When I attach `contract_token` with sudo ... passed in 255.276s
    And I run `ua enable <service> <flag>` as non-root ... passed in 0.755s
    Then I will see the following on stderr ... failed in 0.000s
Assertion Failed: 
Expected: 'This command must be run as root (try using sudo)'
     but: was 'usage: ua [command] [flags]\nua: error: unrecognized arguments: <flag>'

      """
      This command must be run as root (try using sudo)
      """
```

Reply:
@lucasmoura sorry I forgot to push fixes for that.